Dockerfile

```diff
@@ -291,6 +297,21 @@ ARG TAG
 ENV VERSION ${TAG}
 LABEL "alpha.talos.io/version"="${VERSION}"
 ENTRYPOINT ["entrypoint.sh"]
+ONBUILD WORKDIR /initramfs
+ONBUILD ARG RM
+ONBUILD RUN xz -d /usr/install/initramfs.xz \
+    && cpio -idvm < /usr/install/initramfs \
+    && unsquashfs -f -d /rootfs rootfs.sqsh \
+    && for f in ${RM}; do rm -rfv /rootfs$f; done \
+    && rm /usr/install/initramfs \
+    && rm rootfs.sqsh
+ONBUILD COPY --from=customization / /rootfs
+ONBUILD RUN find /rootfs \
```

Comment:
The `find /rootfs` is useful as it prints out the new roofs layout. I found this useful when conforming that the new files I specified actually made it into the image.